postgres-appliance/Dockerfile

```diff
@@ -151,7 +151,8 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                         postgresql-${version}-postgis-${POSTGIS_VERSION%.0} \
                         postgresql-${version}-postgis-${POSTGIS_VERSION%.0}-scripts \
                         postgresql-${version}-repack \
-                        postgresql-${version}-wal2json" \
+                        postgresql-${version}-wal2json \
+                        postgresql-${version}-pgaudit" \
```

Comment:
Please move it after hypopg: https://github.com/zalando/spilo/pull/448/files#diff-82f985fcf1262fb2d3d1843b9e74d9c3R141, so extensions are ordered alphabetically :)

Reply:
ok, done. :)